doc/user/content/integrations/websocket-api.md

```diff
@@ -55,10 +55,27 @@ The WebSocket API provides two modes with slightly different transactional seman
 wss://<MZ host address>/api/experimental/sql
 ```
 
-To authenticate, send an initial text or binary message containing a JSON object with keys:
+To authenticate using a username and password, send an initial text or binary message containing a JSON object:
```

Comment:
I feel like it'd be more JS-y to use untagged enums here? So you'd either show up and say:

```
{ "user": ..., "password": ... }
```

or:

```
{ "token": ... }
```

That's what we do for the simple vs extended protocol too, right?

Reply:
Yeah, this is better. Done.